### DISCUSSION TITLE
How to extend AzureLogAnalyticsWorkspaceResource with CustomTables?

### DISCUSSION THREAD

**Discussion by ryanheath** [microsoft/aspire#5166] [Q&A] [answered]
I'd like to write to a custom table in Azure LogAnalyticsWorkspace.
There is already a workspace created by default.
I want to add custom tables to this workspace.

AzureLogAnalyticsWorkspaceExtensions seems to use OperationalInsightsWorkspaceSku to do its 'magic'.
What could I use to extend this with creation of customtables?

[See AzureLogAnalyticsWorkspaceExtensions in Aspire.Hosting.Azure.OperationalInsights](https://github.com/dotnet/aspire/blob/a41bd1baf4367f6b9394575d75dd995340b4795b/src/Aspire.Hosting.Azure.OperationalInsights/AzureLogAnalyticsWorkspaceExtensions.cs#L44)

**Comment by davidfowl** [comment, marked as answer]
You can do it now in Aspire 9.2 with AddAzureContainerAppEnvironment
